tests/ReflectionTarget.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Brick\Reflection\Tests;
+
+/**
+ * The Reflection Target class.
+ */
+class ReflectionTarget
+{
+    /**
+     * @param string
+     */
+    private $foo;
+
+    /**
+     * @var string $bar
+     */
+    private $bar;
+
+    /**
+     * @var \Exception $barWithType
+     */
+    private $barWithType;
+
+    public function __construct()
+    {
+        $this->foo = 'foo';
+        $this->bar = 'bar';
+    }
+
+    /**
+     * @param  string
```

Comment:
Now this should be `@param string $str`!

```php
/**
 * @param string $str
 *
 * @return string
*/
```

`@param` mentions the parameter name
`@return` does not